packages/react-ui/internal/ResizeDetector/ResizeDetector.tsx

```diff
@@ -7,6 +7,8 @@ import { styles } from './ResizeDetector.styles';
 export interface ResizeDetectorProps {
   onResize?: (event: UIEvent) => void;
   fullHeight?: boolean;
+  className?: React.HTMLAttributes<HTMLElement>['className'];
+  style?: React.HTMLAttributes<HTMLElement>['style'];
```

Comment:
У нас есть тип `HTMLProps`, с помощью которого этот код можно сделать чуточку более читабельным:
```suggestion
  className?: HTMLProps['div']['className'];
  style?: HTMLProps['div']['style'];
```